sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -1297,6 +1301,46 @@ public void testLatestAggregators() throws Exception
     );
   }
 
+  // This test the on-heap version of the AnyAggregator (Double/Float/Long/String)
+  @Test
+  public void testAnyAggregator() throws Exception
```

Comment:
nit: It would probably be worth adding an additional test that tests numeric columns agains `druid.numfoo` table since it contains numeric columns that have null values when run in sql compatible null mode, and also tests for ordering by each 'any' aggregator.

Reply:
done